projects/storefrontstyles/scss/_page-template.scss

```diff
@@ -2,23 +2,23 @@
 
 @import './layout/index';
 
-// TODO (#11607) remove LoginPageTemplate from whitelist
+// TODO (#11607) remove LoginPageTemplate from allowlist
 
-// The $page-template-blacklist can be used to prevent template CSS
+// The $page-template-blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist: () !default;
+$page-template-blocklist: () !default;
 
-// whitelisted page template selectors are processed
-// unless (some) they are blacklisted using `$page-template-blacklist`
-$page-template-whitelist: LandingPage2Template, ContentPage1Template,
+// allowlisted page template selectors are processed
+// unless (some) they are blocklisted using `$page-template-blocklist`
```

Comment:
[nitpick] could you change this to `// unless (some) that are blocklisted using `$page-template-blocklist``